javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1279,4 +1281,18 @@ public static Symbol findFunction(Tree tree, ProcessingEnvironment env) {
         Types javacTypes = Types.instance(ctx);
         return javacTypes.findDescriptorSymbol(((Type) typeOf(tree)).asElement());
     }
+
+    /**
+     * Returns true if {@code tree} is an implicitly typed lambda.
+     *
+     * <p>A lambda expression whose formal type parameters have inferred types is an implicitly
+     * typed lambda. (See JLS 15.27.1)
+     *
+     * @param tree any kind of tree
+     * @return true iff {@code tree} is an implicitly typed lambda.
+     */
+    public static boolean isImplicitlyTypeLambda(Tree tree) {
```

Comment:
`isImplicitlyTypeLambda` -> `isImplicitlyTypedLambda` for consistency.

Reply:
Thanks for catching all my typos!